src/main/java/net/demilich/metastone/game/Player.java

```diff
@@ -1,184 +1,184 @@
-		this.deck = selectedDeck.getCardsCopy();
-		this.setHero(config.getHeroForPlay().createHero());
-		this.deckName = selectedDeck.getName();
-		setBehaviour(config.getBehaviour().clone());
-		setHideCards(config.hideCards());
-	}
-
-	@Override
-	public Player clone() {
-		return new Player(this);
-	}
-
-	public IBehaviour getBehaviour() {
-		return behaviour;
-	}
-
-	public List<Actor> getCharacters() {
-		List<Actor> characters = new ArrayList<Actor>();
-		characters.add(getHero());
-		characters.addAll(getMinions());
-		return characters;
-	}
-
-	public CardCollection getDeck() {
-		return deck;
-	}
-
-	public String getDeckName() {
-		return deckName;
-	}
-
-	public List<Entity> getGraveyard() {
-		return graveyard;
-	}
-
-	public CardCollection getHand() {
-		return hand;
-	}
-
-	public Hero getHero() {
-		return hero;
-	}
-
-	public int getId() {
-		return id;
-	}
-
-	public int getLockedMana() {
-		return lockedMana;
-	}
-
-	public int getMana() {
-		return mana;
-	}
-
-	public int getMaxMana() {
-		return maxMana;
-	}
-
-	public List<Minion> getMinions() {
-		return minions;
-	}
-
-	public String getName() {
-		return "'" + name + "' (" + getHero().getName() + ")";
-	}
-
-	public HashSet<String> getSecrets() {
-		return secrets;
-	}
-	
-	public List<Entity> getSetAsideZone() {
-		return setAsideZone;
-	}
-
-	public GameStatistics getStatistics() {
-		return statistics;
-	}
-
-	public boolean hideCards() {
-		return hideCards && !(behaviour instanceof HumanBehaviour);
-	}
-
-	public void setBehaviour(IBehaviour behaviour) {
-		this.behaviour = behaviour;
-	}
-
-	public void setHero(Hero hero) {
-		this.hero = hero;
-	}
-
-	public void setHideCards(boolean hideCards) {
-		this.hideCards = hideCards;
-	}
-
-	public void setId(int id) {
-		this.id = id;
-	}
-
-	public void setLockedMana(int lockedMana) {
-		this.lockedMana = lockedMana;
-	}
-
-	public void setMana(int mana) {
-		this.mana = mana;
-	}
-
-	public void setMaxMana(int maxMana) {
-		this.maxMana = maxMana;
-	}
-
-	@Override
-	public String toString() {
-		return "[PLAYER " + "id: " + getId() + ", name: " + getName() + ", hero: " + getHero() + "]";
-	}
-
-}
+package net.demilich.metastone.game;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+
+import net.demilich.metastone.game.behaviour.IBehaviour;
+import net.demilich.metastone.game.behaviour.human.HumanBehaviour;
+import net.demilich.metastone.game.cards.CardCollection;
+import net.demilich.metastone.game.decks.Deck;
+import net.demilich.metastone.game.entities.Actor;
+import net.demilich.metastone.game.entities.Entity;
+import net.demilich.metastone.game.entities.heroes.Hero;
+import net.demilich.metastone.game.entities.minions.Minion;
+import net.demilich.metastone.game.logic.CustomCloneable;
+import net.demilich.metastone.game.statistics.GameStatistics;
+import net.demilich.metastone.game.gameconfig.PlayerConfig;
+
+public class Player extends CustomCloneable {

```

Comment:
Nothing, but I thought something did when I added the file. Turns out
nothing did, but things are weird sometimes.
On Apr 15, 2016 11:10 PM, "AkosCz" <notifications@github.com> wrote:

> In src/main/java/net/demilich/metastone/game/Player.java
> <https://github.com/demilich1/metastone/pull/122#discussion_r59962112>:
>
> > +import java.util.HashSet;
> > +import java.util.List;
> > +
> > +import net.demilich.metastone.game.behaviour.IBehaviour;
> > +import net.demilich.metastone.game.behaviour.human.HumanBehaviour;
> > +import net.demilich.metastone.game.cards.CardCollection;
> > +import net.demilich.metastone.game.decks.Deck;
> > +import net.demilich.metastone.game.entities.Actor;
> > +import net.demilich.metastone.game.entities.Entity;
> > +import net.demilich.metastone.game.entities.heroes.Hero;
> > +import net.demilich.metastone.game.entities.minions.Minion;
> > +import net.demilich.metastone.game.logic.CustomCloneable;
> > +import net.demilich.metastone.game.statistics.GameStatistics;
> > +import net.demilich.metastone.game.gameconfig.PlayerConfig;
> > +
> > +public class Player extends CustomCloneable {
>
> What changed in this file? Just white space changes?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/demilich1/metastone/pull/122/files/2d5ff80764e52deba49388465b03e4737cb19927#r59962112>
>